### PR TITLE
ddl: make TestNotifyDDLJob stable

### PR DESCRIPTION
**Pull request by zimulala**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

```
[2021-03-09T09:00:56.914Z] ----------------------------------------------------------------------
[2021-03-09T09:00:56.914Z] FAIL: ddl_worker_test.go:80: testDDLSuite.TestNotifyDDLJob
[2021-03-09T09:00:56.914Z]
[2021-03-09T09:00:56.914Z] ddl_worker_test.go:128:
[2021-03-09T09:00:56.914Z]     c.Fatal("do not get the add index job notification")
[2021-03-09T09:00:56.914Z] ... Error: do not get the add index job notification
[2021-03-09T09:00:56.914Z]
```

### What is changed and how it works?

Proposal: [xxx](url) 

How it Works:

Ensure that the notification is not handled by the worker's "start".

### Check List 

Tests 

- No code

### Release note 

- No release note.

### CODE REVIEW

File: ddl/ddl_worker_test.go [hunk 1]
```diff
@@ -102,6 +102,11 @@ func (s *testDDLSuite) TestNotifyDDLJob(c *C) {
 	)
 	defer d.Stop()
 	getFirstNotificationAfterStartDDL(d)
+	// Ensure that the notification is not handled by worker's "start".
```
Comment (AilinKid) [inline ddl/ddl_worker_test.go:105]:
```suggestion
	// Ensure that the notification is not handled in workers `start` function.
```

**Review by xhebox — APPROVED** [review]
/lgtm

**Pull request merged**